drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -187,51 +187,81 @@ function DragAndDropTemplates(configuration) {
-        var properties = { attributes: { 'aria-live': 'polite' } };
+        var properties = {attributes: {'aria-live': 'polite'}};
         return (
             h('section.feedback', properties, [
-                h(
-                    'button.reset-button.unbutton.link-button',
-                    { style: { display: reset_button_display }, attributes: { tabindex: 0 }, 'aria-live': 'off'},
-                    gettext('Reset problem')
-                ),
-                h('h3.title1', { style: { display: feedback_display } }, gettext('Feedback')),
-                h('p.message', { style: { display: feedback_display }, innerHTML: ctx.feedback_html })
+                h('h3.title1', {style: {display: feedback_display}}, gettext('Feedback')),
+                h('p.message', {style: {display: feedback_display}, innerHTML: ctx.feedback_html})
             ])
         );
     };
 
-    var keyboardHelpTemplate = function(ctx) {
-        var dialog_attributes = { role: 'dialog', 'aria-labelledby': 'modal-window-title' };
-        var dialog_style = {};
+    var keyboardHelpPopupTemplate = function(ctx) {
         return (
-            h('section.keyboard-help', [
-                h('button.keyboard-help-button.unbutton.link-button', { attributes: { tabindex: 0 } }, gettext('Keyboard Help')),
-                h('div.keyboard-help-dialog', [
-                    h('div.modal-window-overlay'),
-                    h('div.modal-window', { attributes: dialog_attributes, style: dialog_style }, [
-                        h('div.modal-header', [
-                            h('h2.modal-window-title', gettext('Keyboard Help'))
-                        ]),
-                        h('div.modal-content', [
-                            h('p', gettext('You can complete this problem using only your keyboard.')),
-                            h('ul', [
-                                h('li', gettext('Use "Tab" and "Shift-Tab" to navigate between items and zones.')),
-                                h('li', gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.')),
-                                h('li', gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" to drop the item on the current zone.')),
-                                h('li', gettext('Press "Esc" if you want to cancel the drop operation (for example, to select a different item).')),
-                            ])
-                        ]),
-                        h('div.modal-actions', [
-                            h('button.modal-dismiss-button', gettext("OK"))
+            h('div.keyboard-help-dialog', [
+                h('div.modal-window-overlay'),
+                h('div.modal-window', {attributes: {role: 'dialog', 'aria-labelledby': 'modal-window-title'}}, [
+                    h('div.modal-header', [
+                        h('h2.modal-window-title#modal-window-title', gettext('Keyboard Help'))
```

Comment:
This caused a non-unique ID when 2 problems are on the same page. It is very easy to look for these sort of issues using Chrome and the aXe developer tool plugin.

![image](https://cloud.githubusercontent.com/assets/484484/17351422/545721dc-58fc-11e6-94b2-8e9c9f84db4f.png)